app/controllers/inbox/meetings/new.js

```diff
@@ -0,0 +1,42 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { dropTask } from 'ember-concurrency';
+import { inject as service } from '@ember/service';
+
+export default class InboxMeetingsNewController extends Controller {
+  @service router;
+
+  get meeting() {
+    return this.model.meeting;
+  }
+
+  @dropTask
+  * saveMeetingTask(event) {
+    event.preventDefault();
+
+    let bestuursorgaan = yield this.meeting.bestuursorgaan;
+    if (!bestuursorgaan) {
+      this.meeting.errors.add('bestuursorgaan', 'inbox.meetings.new.meeting.errors.administrativeBody.required');
+    }
+
+    if (this.meeting.isValid) {
+      yield this.meeting.save();
+      this.router.replaceWith('meetings.edit', this.meeting.id);
+    }
+  }
+
+  @action
+  updateAdministrativeBody(administrativeBody) {
+    this.meeting.bestuursorgaan = administrativeBody;
+    this.meeting.errors.remove('bestuursorgaan');
+  }
+
+  @action
+  cancelMeetingCreation() {
+    // Prevent unwanted calls due to focus trap...
+    if (this.meeting.isNew && !this.meeting.isDeleted) {
```

Comment:
the ember-focus-trap double execution issue bit me again. I PR'd a fix for it a while ago, but that only fixes the issue if you also change the `@modalOpen` argument of the AuModal component. Since it's statically set to `true` in this route, the issue still occurs. I think this is a deeper issue in the focus-trap modifier itself but I haven't figured it out the fix yet.

Reply:
https://github.com/appuniversum/ember-appuniversum/pull/111 should fix this issue.

Reply:
Removed the logic since it's no longer needed now.